bundles/org.openhab.ui/web/src/pages/settings/things/channel/channel-edit.vue

```diff
@@ -38,10 +42,12 @@
 
 <script>
 import ConfigSheet from '@/components/config/config-sheet.vue'
+import ClipboardIcon from '@/components/util/clipboard-icon'
```

Comment:
Speaking of which - I've made an habit of specifying the extension when importing (except if it's .js and except in `@/assets/definitions/widgets` because the node loader is different from webpack and requires them to generate the docs).

```suggestion
import ClipboardIcon from '@/components/util/clipboard-icon.vue'
```